samples/pytorch_onnx_inference/README.md

```diff
@@ -1,18 +1,58 @@
-## Machine Learning Inference sample with Mystikos
+# Machine Learning Inference sample in Python running in Mystikos
 
-### Functionality 
-This sample downloads AlexNet in `.pt` format and exports it to `.onnx`
+This sample demonstrates how a machine learning model can be donwloaded into an enclave and
```

Comment:
The model is not downloaded into the enclave. It's downloaded in the host and then added into the rootfs with appbuilder.

Reply:
thanks. fixed.